Index: syslog-ng-3.5.6/configure.ac
===================================================================
--- syslog-ng-3.5.6.orig/configure.ac	2014-08-04 13:55:33.000000000 +0200
+++ syslog-ng-3.5.6/configure.ac	2015-09-08 16:01:51.863791323 +0200
@@ -1055,7 +1055,7 @@
 fi
 
 if test "x$enable_systemd" = "xyes"; then
-	PKG_CHECK_MODULES(libsystemd_daemon, libsystemd-daemon >= 31,enable_systemd="yes",enable_systemd="no")
+	PKG_CHECK_MODULES(libsystemd, libsystemd >= 209,enable_systemd="yes",enable_systemd="no")
 	if test "x$with_systemdsystemunitdir" = "xyes"; then
 		# no arguments, just --with-systemdsystemunitdir
 		systemdsystemunitdir=`$PKG_CONFIG --variable=systemdsystemunitdir systemd`
Index: syslog-ng-3.5.6/lib/Makefile.am
===================================================================
--- syslog-ng-3.5.6.orig/lib/Makefile.am	2014-08-04 13:55:33.000000000 +0200
+++ syslog-ng-3.5.6/lib/Makefile.am	2015-09-08 16:01:03.759425142 +0200
@@ -7,7 +7,7 @@
 include lib/template/Makefile.am
 
 lib_LTLIBRARIES				+= lib/libsyslog-ng.la
-lib_libsyslog_ng_la_LIBADD		= @CORE_DEPS_LIBS@ $(libsystemd_daemon_LIBS)
+lib_libsyslog_ng_la_LIBADD		= @CORE_DEPS_LIBS@ $(libsystemd_LIBS)
 lib_libsyslog_ng_la_LDFLAGS		= -no-undefined -release @VERSION@
 
 lib_test_subdirs			= lib_filter lib_logproto lib_parser lib_rewrite lib_template
Index: syslog-ng-3.5.6/modules/afsocket/Makefile.am
===================================================================
--- syslog-ng-3.5.6.orig/modules/afsocket/Makefile.am	2014-08-04 13:55:33.000000000 +0200
+++ syslog-ng-3.5.6/modules/afsocket/Makefile.am	2015-09-08 16:01:27.327604544 +0200
@@ -33,12 +33,12 @@
 	modules/afsocket/afsocket-plugin.c
 
 modules_afsocket_libafsocket_notls_la_CPPFLAGS	=	\
-	$(AM_CPPFLAGS) $(libsystemd_daemon_CFLAGS)	\
+	$(AM_CPPFLAGS) $(libsystemd_CFLAGS)	\
 	-I${top_srcdir}/modules/afsocket		\
 	-I${top_builddir}/modules/afsocket
 modules_afsocket_libafsocket_notls_la_LIBADD	=	\
 	$(MODULE_DEPS_LIBS) $(LIBNET_LIBS)		\
-	$(LIBWRAP_LIBS) $(libsystemd_daemon_LIBS)
+	$(LIBWRAP_LIBS) $(libsystemd_LIBS)
 modules_afsocket_libafsocket_notls_la_LDFLAGS	  =	\
 	$(MODULE_LDFLAGS)
 modules_afsocket_libafsocket_notls_la_DEPENDENCIES=	\
Index: syslog-ng-3.5.6/modules/system-source/Makefile.am
===================================================================
--- syslog-ng-3.5.6.orig/modules/system-source/Makefile.am	2014-08-04 13:55:33.000000000 +0200
+++ syslog-ng-3.5.6/modules/system-source/Makefile.am	2015-09-08 16:01:15.407513809 +0200
@@ -3,9 +3,9 @@
 modules_system_source_libsystem_source_la_SOURCES	=	\
 	modules/system-source/system-source.c
 modules_system_source_libsystem_source_la_CPPFLAGS	=	\
-	$(AM_CPPFLAGS) $(libsystemd_daemon_CFLAGS)
+	$(AM_CPPFLAGS) $(libsystemd_CFLAGS)
 modules_system_source_libsystem_source_la_LIBADD	=	\
-	$(MODULE_DEPS_LIBS) $(libsystemd_daemon_LIBS)
+	$(MODULE_DEPS_LIBS) $(libsystemd_LIBS)
 modules_system_source_libsystem_source_la_LDFLAGS	=	\
 	$(MODULE_LDFLAGS)
 modules_system_source_libsystem_source_la_DEPENDENCIES	=	\
